### ISSUE TITLE
disrupt:decommission: some threads aren't cleaned in destroying node

### ISSUE THREAD

**Issue by kongove** [scylladb/scylla-cluster-tests#262]
* Error description:

I saw some error "Error 24: Too many open files" in [longevity-gce-1.7 job](http://jenkins.cloudius-systems.com/job/scylla-1.7-longevity-gce/label=gce/24/console), they occurred after job run `3` days.

I just workarounded this issue by adjust the 'open files limit' for avocado process.

```
# original limit is 4096, I just increased it to 10000, the job will finish after 3 days, it should be enough.
prlimit -n10000 -p pid_of_process
```

* Problem in nemesis: 

There are many opened database.log, I found the journal_thread isn't stop when we destroy target node in disrupt decommission test.

**Comment by kongove** [comment]
There are also some SSH connection for checking coredump, it also costs the open files.

both journal thread and getbacktrace(coredump) thread are all dead loop by `while True`, we should stop them when we destroy node.

**Comment by kongove** [comment]
Solved by PR #263 #266
